client/components/board/boardFactory.js

```diff
@@ -0,0 +1,48 @@
+angular.module('OpenEXP')
+    .factory('boardFactory', () => {
+        const OpenBCIBoard = require('openbci-sdk');
+        const ourBoard = new OpenBCIBoard.OpenBCIBoard();
+
+        var findPorts = () => {
+            return new Promise((res, rej) => {
+                ourBoard.autoFindOpenBCIBoard((portName, ports) => {
+                    if(portName) {
+                        res(connect());
+                    } else {
+                        res(ports);
+                    }
+                })
+            });
+        };
+        var storage = {};
+
+        var connect = (portName) => {
+            ourBoard.connect(portName)
+                .then(boardSerial => {
+                    ourBoard.on('ready',function() {
+                        /** Start streaming, reading registers, what ever your heart desires  */
+                    });
+                }).catch(function(err) {
+                /** Handle connection errors */
+            });
+        };
+
+        var observer = function(changes) {
```

Comment:
is there a reason you use the function() syntax instead of the => ?  Just trying to educate myself :)